test/helpers/components/dynakube/dynakube.go

```diff
@@ -27,9 +29,16 @@ const (
 	defaultName = "dynakube"
 )
 
-func Install(builder *features.FeatureBuilder, level features.Level, secretConfig *tenant.Secret, testDynakube dynakubev1beta2.DynaKube) {
-	Create(builder, level, secretConfig, testDynakube)
-	VerifyStartup(builder, level, testDynakube)
+func Install(builder *features.FeatureBuilder, level features.Level, secretConfig *tenant.Secret, testDynakube dynakubev1beta2.DynaKube, usesOldVersion bool) {
+	if usesOldVersion {
+		dkv1beta1 := dynakubev1beta1.DynaKube{}
+		dkv1beta1.ConvertFrom(&testDynakube)
+		CreateV1Beta1(builder, level, secretConfig, dkv1beta1)
+		VerifyStartupV1Beta1(builder, level, dkv1beta1)
+	} else {
+		Create(builder, level, secretConfig, testDynakube)
+		VerifyStartup(builder, level, testDynakube)
+	}
 }
```

Comment:
you could just introduce an `InstallAsOld` or something, so you only change a single package and not every

```
func InstallAsOld(builder *features.FeatureBuilder, level features.Level, secretConfig *tenant.Secret, testDynakube dynakubev1beta2.DynaKube) {
	dkv1beta1 := dynakubev1beta1.DynaKube{}
	dkv1beta1.ConvertFrom(&testDynakube)
	CreateV1Beta1(builder, level, secretConfig, dkv1beta1)
	VerifyStartupV1Beta1(builder, level, dkv1beta1)
}
```

The name `InstallAsOld` is not my greatest, so pls try to change it 😅

Reply:
I had this idea also yesterday in the evening, on it :+1:

Reply:
done in https://github.com/Dynatrace/dynatrace-operator/pull/3205/commits/0ce7b62c82226527c1db1832f2fb808d8d81983a

Please give me feedback for the naming 😄